tably.py

```diff
@@ -78,16 +108,21 @@ def run(self):
         if self.preamble:
             all_tables.insert(0, PREAMBLE)
             all_tables.append('\\end{document}\n')
-        else:
-            all_tables.insert(0, '\n% \\usepackage{booktabs} % move this to '
-                                 'preamble and uncomment')
 
         final_content = '\n\n'.join(all_tables)
         if self.outfile:
             try:
-                save_content(final_content, self.outfile)
+                save_content(final_content, self.outfile, self.replace)
             except FileNotFoundError:
                 print('{} is not a valid/known path. Could not save there.'.format(self.outfile))
+        elif self.separate_outfiles is not None: # if -oo is passed
+            outs = self.separate_outfiles
+            if len(outs) == 0:
+                outs = [ file[:-4]+'.tex' if file[-4:]=='.csv' else file+'.tex' for file in self.files]
```

Comment:
What if there is a file with an extension which is not `.csv`? This would make it `filename.ext.tex`.